src/test/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2IntegrationTest.java

```diff
@@ -19,9 +19,15 @@
     //TODO: this requires Broad server; repalce with the mini reference in large test files directory
     private static final File hg19_REFERENCE = new File("/seq/references/Homo_sapiens_assembly19/v1/Homo_sapiens_assembly19.fasta");
 
+    //TODO: note this test works if you run it locally and mount /seq/references
+    //TODO: it's commented out because Travis will fail, but it's useful to have for development until
+    //TODO: real imntegration tests come
+    //TODO: currently, results look reasonable on what is admittedly a pretty tiny test
+
     /*
      * Run the tool
      */
+    /*
     @Test
```

Comment:
minor detail but how about replacing `@test` with the `@ignore` testNG annotation instead of comment?